tests/test_quickbooks_pagination.py

```diff
@@ -29,6 +25,7 @@ def get_properties(self):
 
 
     def test_run(self):
+        """Executing run_test with different page_size values for different streams"""
```

Comment:
Mention the expectations in the docstring.

Reply:
done